### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by rvolosatovs**
#### Summary

Closes https://github.com/TheThingsIndustries/release-notes/issues/3

#### Changes

- Add changelog as defined at https://keepachangelog.com/en/1.0.0/
- Remove "Release notes" section from PR template
- Reference changelog in our github releases from now on

#### Notes for Reviewers

Formatted release notes according to the format defined at https://keepachangelog.com/en/1.0.0/, i.e. imperative tense and `.` at the end.
You can see https://github.com/rvolosatovs/lorawan-stack/releases/tag/v3.2.1 for an example. Note, that `Release notes` link there does not take you to the actual section in the changelog, since `v3.2.1` does not exist there. Normally the link points to the relevant section in the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [3.2.1] - 2019-10-09
```
[the remark this answers is not in the document]
Comment (rvolosatovs) [reply CHANGELOG.md]: Oops

File: .travis.yml [hunk 2]
```diff
@@ -137,7 +137,7 @@ after_success:
     docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
     snapcraft login --with snapcraft.login
     if [[ ! -z "$TRAVIS_TAG" && "$TRAVIS_TAG" =~ ^v3\.[0-9]+\.[0-9]+ ]]; then
-      GO111MODULE=on go run github.com/goreleaser/goreleaser --release-notes <(go run github.com/TheThingsIndustries/release-notes -owner TheThingsNetwork -repo lorawan-stack -id "release-notes" -head $(git rev-parse HEAD) | sort)
+      GO111MODULE=on go run github.com/goreleaser/goreleaser --release-notes <(printf "[Release notes](https://github.com/TheThingsIndustries/lorawan-stack/blob/${TRAVIS_TAG}/CHANGELOG.md#$(echo ${TRAVIS_TAG} | sed "s/v\([1-9]\+\)\.\([1-9]\+\)\.\([1-9]\+\)/\1\2\3---$(date +%Y-%m-%d)/"))")
```
Comment (htdvisser) [inline .travis.yml]: Shouldn't we just include the CHANGELOG file as release notes instead of linking to the file on Github?

Comment (rvolosatovs) [reply .travis.yml]: Do you mean adding the relevant section from `CHANGELOG.md` to the release?
We probably should do that later, but just having a link should be good enough for now.

Comment (johanstokking) [reply .travis.yml]: This link goes to the TTI repo btw, should be TTN

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -256,6 +256,37 @@ Adding translations of messages to other languages than English is a matter of a
 
 Translations of frontend messages are located in `pkg/webui/locales`.
 
+## Release notes
+We maintain a changelog at `CHANGELOG.md` using a format based on [Keep a Changelog].
+Any notes that we need to include in the Release Notes for the next release should be added under the `Unreleased` section.
```
Comment (johanstokking) [inline CONTRIBUTING.md]: No line breaks without empty lines. Keep things on a single line and enable word-wrap, or put an empty line to create a paragraph.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -256,6 +256,37 @@ Adding translations of messages to other languages than English is a matter of a
...
+
+Please consult documentation at [Keep a Changelog] on how to write these notes.
+Some key points:
+- Notes are formatted as bullet points, written in imperative tense and finish with a dot(`.`).
+- There are 6 possible types of changes, always listed in following order:
```
Comment (johanstokking) [inline CONTRIBUTING.md]: six

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -256,6 +256,37 @@ Adding translations of messages to other languages than English is a matter of a
 
 Translations of frontend messages are located in `pkg/webui/locales`.
 
+## Release notes
+We maintain a changelog at `CHANGELOG.md` using a format based on [Keep a Changelog].
+Any notes that we need to include in the Release Notes for the next release should be added under the `Unreleased` section.
+
+Please consult documentation at [Keep a Changelog] on how to write these notes.
+Some key points:
+- Notes are formatted as bullet points, written in imperative tense and finish with a dot(`.`).
```
Comment (johanstokking) [inline CONTRIBUTING.md]: Space before parentheses

**Pull request merged**